analyzer/requirements_py/dev/requirements.txt

```diff
@@ -1,3 +1,4 @@
+setuptools-rust==0.12.1
```

Comment:
This is needed for twine which can be found in another requirements file:https://github.com/Ericsson/codechecker/blob/master/web/requirements_py/dev/requirements.txt#L19
Please insert this line to this file below the twine dependency.